mediaqueries-5/Overview.bs

```diff
@@ -478,6 +478,36 @@ Detecting the desire for less motion on the page: the 'prefers-reduced-motion' f
 
 	</dl>
 
+<h3 id="prefers-reduced-transparency">
+Detecting the desire to reduce the use of transparent or translucent layering effects on the page: the 'prefers-reduced-transparency' feature</h3>
```

Comment:
Transparency is a boolean concept; translucency and opacity are not. If the feature name contains `transparency` for the sake of easy vs more correct language, then the docs can easily contain `translucent` and/or `opacity` if authors are search the docs for those (more correct) terms.

But - as long as @cookiecrook hadn't had more specifics in mind - skipping the `layering effects` vs just `page` 👍 

> Detecting the desire for reduced transparency or translucency on the page: the 'prefers-reduced-transparency' feature